os/drivers/audio/alc5658scripts.h

```diff
@@ -29,370 +29,281 @@
-	{ALC5658_ADDA_CLK, 0x1770, 0},	/* I2Sprediv1 = 2, bclk_ms2 = 16bits(32FS), I2Sprediv2 = 7, something reserved, dac/adc 128Fs */
+	{ALC5658_MICBIAS2, 0x0280, 0},	/* ??? Enable ALL int CLK, even more than in documentation */
+	{ALC5658_CLOCK, 0x8000, 0},	/* RC CLK, No dividers */
+	{ALC5658_I2S1_PORT_CTRL, 0x0000, 0},	/* Master, off/normal/I2S...  16bit */
+	{ALC5658_ADCDAC_CLOCK, 0x1770, 0},	/* I2Sprediv1 = 2, bclk_ms2 = 16bits(32FS), I2Sprediv2 = 7, something reserved, dac/adc 128Fs */
 	{ALC5658_0091, 0x0C16, 0},
-	{ALC5658_PWR_MNG3, 0xAA7E, 60},	/* VREF1 on, fast VREF1, VREF2 on, fast VREF2, MBIAS on, LOUT off, MBIAS bandgap off, HP L/R ON, HPamp x5, LDO1 out 1.2V */
-	{ALC5658_PWR_MNG3, 0xFE7E, 50},	/* VREF1 on, SLOW VREF1, VREF2 on, SLOW VREF2, MBIAS on, LOUT off, MBIAS bandgap off, HP L/R ON, HPamp x5, LDO1 out 1.2V */
-	{ALC5658_PWR_MNG5, 0x0004, 0},	/* LDO2 ON */
-	{ALC5658_PWR_MNG2, 0x0400, 50},	/* pow_dac_stereo1_filter ON */
-	{ALC5658_PWR_MNG1, 0x0080, 10},	/* Pow_ldo_dacref ON */
-	{ALC5658_ADC_2_DAC_MXR, 0x8080, 0},	/* Mu_stereo1_adc_mixer_l/r MUTE */
-	{ALC5658_DAC_STR_MXR, 0xAAAA, 0},	/* Default, mute all */
+	{ALC5658_MANAGEMENT3, 0xAA7E, 60},	/* VREF1 on, fast VREF1, VREF2 on, fast VREF2, MBIAS on, LOUT off, MBIAS bandgap off, HP L/R ON, HPamp x5, LDO1 out 1.2V */
+	{ALC5658_MANAGEMENT3, 0xFE7E, 50},	/* VREF1 on, SLOW VREF1, VREF2 on, SLOW VREF2, MBIAS on, LOUT off, MBIAS bandgap off, HP L/R ON, HPamp x5, LDO1 out 1.2V */
+	{ALC5658_MANAGEMENT5, 0x0004, 0},	/* LDO2 ON */
+	{ALC5658_MANAGEMENT2, 0x0400, 50},	/* pow_dac_stereo1_filter ON */
+	{ALC5658_MANAGEMENT1, 0x0080, 10},	/* Pow_ldo_dacref ON */
+	{ALC5658_ADC_2_MXR, 0x8080, 0},	/* Mu_stereo1_adc_mixer_l/r MUTE */
+	{ALC5658_DAC_ST_MXR, 0xAAAA, 0},	/* Default, mute all */
 	{ALC5658_DAC_SRC, 0x0000, 0},	/* Default, no mixers (direct) */
-	{ALC5658_HP_AMP, 0x0009, 50},	/* en_out_hp - OFF, pow_pump_hp - ON, pow_capless - ON */
-	{ALC5658_PWR_MNG1, 0x0F80, 50},	/* pow_dac1_l/r, pow_dac2_l/r, Pow_ldo_dacref - ON */
+	{ALC5658_HP, 0x0009, 50},	/* en_out_hp - OFF, pow_pump_hp - ON, pow_capless - ON */
+	{ALC5658_MANAGEMENT1, 0x0F80, 50},	/* pow_dac1_l/r, pow_dac2_l/r, Pow_ldo_dacref - ON */
 	{ALC5658_0091, 0x0E16, 50},
 	{ALC5658_0040, 0x0505, 0},
-	{ALC5658_PWR_MNG5, 0x0184, 0},	/* ???NonDescribed,  LDO2 ON */
-	{ALC5658_0x013C, 0x3C05, 0},
-	{ALC5658_0x01DF, 0x2cc1, 6},
-	{ALC5658_0x01DE, 0x5100, 6},
-	{ALC5658_0x01E4, 0x0014, 6},
-	{ALC5658_0x01DE, 0xd100, 30},
-	{ALC5658_0x01DF, 0x20C1, 0},
-	{ALC5658_0x01DF, 0x2CC1, 0},
-	{ALC5658_0x01DE, 0x4900, 0},
-	{ALC5658_0x01E4, 0x0016, 0},
-	{ALC5658_0x01DE, 0xC900, 250},
-	{ALC5658_0x01DF, 0x2CC1, 0},
-	{ALC5658_HPOUT_MUTE, 0x0000, 0},	/* UNMUTE HP Output */
-	{ALC5658_0x01DE, 0x4500, 0},
-	{ALC5658_0x01E4, 0x001F, 0},
-	{ALC5658_0x01DE, 0xC500, 600},
-	{ALC5658_0x01E4, 0x0000, 0},
-	{ALC5658_0x01DF, 0x20C0, 0},
+	{ALC5658_MANAGEMENT5, 0x0184, 0},	/* ???NonDescribed,  LDO2 ON */
```

Comment:
what is  "???" ?
plese check and then modify that comment.

Reply:
@jsdosa I did not change the comment in the script. I do not know the exact meaning of script comments. Sorry.

Reply:
Okay. But that part has to be erased.

Reply:
please remove that.

Reply:
you mean that you want to delete an indefinite comment, such as '???' ?

Reply:
yes, It is the comment to be erased.